app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+import shutil
+
+
+def copy_file(command: str) -> str:
+    file_1, file_2 = command.split()[1], command.split()[2]
+    with open(file_1, "r") as file_in, open(file_2, "w") as file_out:
+        if file_in == file_out:
+            pass
+        return shutil.copyfile(file_1, file_2)
```

Comment:
No need shututil use standard libraries

Reply:
as far as I know, using side libraries is the proper pythonic way to write code, why not? Why use tons of code if it can be written in one line?